core/src/main/scala/org/broadinstitute/dsde/rawls/entities/local/LocalEntityExpressionValidator.scala

```diff
@@ -17,11 +18,11 @@ class LocalEntityExpressionValidator(dataSource: SlickDataSource)(implicit prote
                                     gatherInputsResult: GatherInputsResult,
                                     allowRootEntity: Boolean): Future[ValidatedMethodConfiguration] = {
 
-    dataSource.inTransaction { parser =>
+    Future {
       val inputsToParse = gatherInputsResult.processableInputs map { mi => (mi.workflowInput.getName, AttributeString(mi.expression)) }
       val (emptyOutputs, outputsToParse) = methodConfiguration.outputs.partition { case (_, expr) => expr.value.isEmpty }
 
-      val parsed = parser.parseMCExpressions(inputsToParse.toMap, outputsToParse, allowRootEntity)
+      val parsed = parseMCExpressions(inputsToParse.toMap, outputsToParse, allowRootEntity)
```

Comment:
moved `parseMCExpressions` from now deleted `ExpressionParser` and we no longer need database stuff

Reply:
Since we aren't using database stuff anymore, this should be easier to share with the upcoming TDR stuff since I'm envisioning that this part of validation will look very similar and will instead use visitors that also check entity lookup nodes against the TDR snapshot schema